clients/privacy-center/cypress/e2e/consent-i18n.cy.ts

```diff
@@ -1,125 +1,361 @@
+/* eslint-disable spaced-comment */
 import { FidesOptions, PrivacyExperience, PrivacyNotice } from "fides-js";
 import { stubConfig } from "../support/stubs";
 
-describe("Consent i18n", () => {
```

Comment:
Warning: there's a lot of red/green in this file. I refactored it to avoid duplication because I was copy-pasta'ing too many assertions between English/Spanish and it was going to mess up somewhere... so you should ignore most of the diffs and focus on the TCF tests that I'll highlight below.